examples/Boost-thread/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# download boost
+hunter_add_package(Boost COMPONENTS system thread)
```

Comment:
Ok. But since static linking we won't get the link error if this_thread::id is not depends on librt. I don't know this.